MorphicServer/Endpoint.cs

```diff
@@ -102,6 +102,17 @@ public abstract class Endpoint
         private static readonly Histogram histogram = Metrics.CreateHistogram(histo_metric_name,
             "HTTP Request Duration",
             labelNames);
+
+        public static string GetPathTemplate(Type T)
```

Comment:
There's a `Type` extension method in `Path.cs` that does this

````
type.GetRoutePath()
````

Reply:
Thanks! Done.